types/digibyte/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for digibyte 0.14
+// Type definitions for digibyte 0.15
```

Comment:
This version is meant to reflect the version on npm, and should not be manually changed otherwise. I am going to revert it back to 0.14.

Reply:
Thanks @DanielRosenwasser, nice info